src/index.js

```diff
@@ -0,0 +1,40 @@
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Header} from './Header.js'
+import {Burger} from './Burger.js'
+import './index.css';
+function Main(){
```

Comment:
Make a main.js file and refactor this logic.

Reply:
Done